src/main/java/menion/android/whereyougo/gui/dialog/NoPasswordDialogFragment.java

```diff
@@ -0,0 +1,52 @@
+package menion.android.whereyougo.gui.dialog;
+
+import android.app.AlertDialog;
+import android.app.Dialog;
+import android.content.Context;
+import android.content.DialogInterface;
+import android.os.Bundle;
+
+import androidx.fragment.app.DialogFragment;
+
+import menion.android.whereyougo.R;
+
+public class NoPasswordDialogFragment extends DialogFragment {
+
+    private NoPasswordDialogListener listener;
+
+    public interface NoPasswordDialogListener {
+        public void onPositiveClick(DialogFragment dialog);
```

Comment:
From AndroidStudio:
modifier** 'public' is redundant for interface method

Reply:
Fixed by c1bf2e3